### PR TITLE
writing simulation to csv file every second

### PR DESCRIPTION
**Pull request by AndiReich**

### CODE REVIEW

File: src/utils/file_utils.gd [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	self.file.store_csv_line(["cell_id", "cell_type", "cell_pos_x", "cell_pos_y", "antigen_code", "time"])
+
+func save_cell_state_to_csv(cell: Cell):
+	var time = Time.get_datetime_string_from_system()
+	var id = cell.get_instance_id()
+	var type = Cell.TYPES.find_key(cell.cell_state_handler.cell_type).to_lower()
+	var pos_x = cell.position.x
+	var pos_y = cell.position.y
+	var antigen_code = cell.cell_state_handler.color_code
+	if type == "macrophage" || type == "bcell" || type == "thelpercellt4":
```
Comment (FelixBangerter) [inline src/utils/file_utils.gd:24]: Is is superfluous since you set the `color_code` to `-1` in the `init` functions of the respective classes, or not?

**Review by FelixBangerter — CHANGES_REQUESTED** [review]
Looks good to me and can be merged if the single comment below has been addressed.

File: src/utils/file_utils.gd (hunk 1 above)
Comment (AndiReich) [reply src/utils/file_utils.gd:24]: No this didnt solve it. Its probably set somewhere else.

Comment (FelixBangerter) [reply src/utils/file_utils.gd:24]: Then that's fine.

Comment (AndiReich) [reply src/utils/file_utils.gd:24]: The correct way would be to reset it to -1. But somewhere else its set to 0.

**Review by FelixBangerter — APPROVED** [review]

**Pull request merged**